### PR TITLE
variants/protectli_vp46xx: update for single binary

### PR DESCRIPTION
**Pull request by mkopec**
I guess this should remain a draft until v1.1.0 is released

### CODE REVIEW

File: docs/variants/protectli_vp46xx/building-manual.md [hunk 1]
```diff
@@ -50,24 +47,15 @@ VP4670. Versions v1.0.17 and older do not support VP4650 and VP4670 at all.
...
-        ```
+1. Build the firmware v1.1.0 or newer:
 
-    + for VP4670
-
-        ```bash
-        ./build.sh vp4670
-        ```
+    ```bash
+    ./build.sh vp46xx
```
Comment (miczyg1) [inline docs/variants/protectli_vp46xx/building-manual.md:57]: 1.1.0 is alredy out and uses old procedure. So it should be for firmware newer than v1.1.0

File: docs/variants/protectli_vp46xx/building-manual.md [hunk 2]
```diff
@@ -50,24 +47,15 @@ VP4670. Versions v1.0.17 and older do not support VP4650 and VP4670 at all.
...
+    ```
 
 The resulting coreboot image will be placed in the coreboot directory as
-`protectli_vault_cml_<version>_vp4630_vp4650.rom` or
-`protectli_vault_cml_<version>_vp4670.rom` respectvely.
+`protectli_vault_cml_<version>_vp46xx.rom`.
```
Comment (miczyg1) [inline docs/variants/protectli_vp46xx/building-manual.md:61]: Place the message under the respective build procedure and keep the old naming there.

File: docs/variants/protectli_vp46xx/building-manual.md [hunk 3]
```diff
@@ -18,20 +18,17 @@ VP4630, VP4650 and VP4670.
 > FSP, and ME. Currently, access to them is restricted to the OEM (Protectli) via
 > a private repository.
 
-Since version v1.0.18 VP4630 and VP4650 use different configuration file than
-VP4670. Versions v1.0.17 and older do not support VP4650 and VP4670 at all.
-
```
Comment (miczyg1) [inline docs/variants/protectli_vp46xx/building-manual.md:23]: Please revert

File: docs/variants/protectli_vp46xx/building-manual.md (hunk 1 above)
Comment (mkopec) [reply docs/variants/protectli_vp46xx/building-manual.md]: Updated in a86d5c3aa62b89e7317fa821fac81c05af9626cb

File: docs/variants/protectli_vp46xx/building-manual.md (hunk 2 above)
Comment (mkopec) [reply docs/variants/protectli_vp46xx/building-manual.md]: Added back the older instructions as tabs in a86d5c3aa62b89e7317fa821fac81c05af9626cb

File: docs/variants/protectli_vp46xx/building-manual.md (hunk 3 above)
Comment (mkopec) [reply docs/variants/protectli_vp46xx/building-manual.md]: Added back in 2d72972c1bd0f7ac4b3576c8c9a702d54867efcf

**Comment by mkopec** [comment]
Added the previous build instructions as content tabs @miczyg1 . Marking as ready for review, as older variants can be built using these instructions.

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**